mission/functions/core/ui/taskroster/fn_tr_listboxtask_select.sqf

```diff
@@ -37,11 +37,15 @@ if (isNil "_coords") then {_coords = [0,0,0]};
 private _distance = round (getPos player distance _coords);
 private _distance_type = "m";
 if(count(str (round _distance)) > 3)then{_distance = (_distance / 1000) toFixed 1; _distance_type = "km";};
-VN_TR_MISSIONSHEET_COORDS_CTRL ctrlSetText format["%1 %2", _distance, _distance_type];
+VN_TR_ACTIVETASKS_RHS_COORDS_CTRL ctrlSetText format["%1 %2", _distance, _distance_type];
 
 VN_TR_MISSION_MAP_CTRL ctrlMapAnimAdd [0, 0.25, _coords];
-ctrlMapAnimCommit VN_TR_MISSION_MAP_CTRL;
+ctrlMapAnimCommit VN_TR_ACTIVETASKS_RHS_MAP_CTRL;
+
+VN_TR_ACTIVETASKS_RHS_NAME_CTRL ctrlSetText (taskDescription _task)#1;
+VN_TR_ACTIVETASKS_RHS_DESC_CTRL ctrlSetText (taskDescription _task)#0;
+
+// set the selected subtask as active
+call vn_mf_fnc_tr_mission_setActive;
```

Comment:
once the RHJS page loads with the sub/child tasks, set the currently selected one as active.

if a sub task has already been selected before, then it should be selected again when this page loads, avoiding the case where opening this page resets the selected task.